### ISSUE TITLE
Feature query: How to load the graph(model) only once and then give series of inputs

### ISSUE THREAD

**Issue by serendipity24** [tensorflow/tensorflow#13835]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:No. Running the Simple Audio recognition network tutorial
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 14.04 
-**TensorFlow installed from (source or binary)**: Source
- **TensorFlow version (use command below)**: 1.3
- **Python version**: 3.6
- **Bazel version (if compiling from source)**: 0.6.0

### Describe the problem
I am a tensorflow and neural network novice. I was able to run the [Simple audio recognition](https://www.tensorflow.org/versions/master/tutorials/audio_recognition) network. For a given input wavefile the model obtained gives the result in approx. 5 seconds. I want reduce this time to 100s of milliseconds. From the issue https://github.com/tensorflow/tensorflow/issues/11618, there seems to be a way to do this using the graph. 

I used the graph optimisation tool but there wasn't a considerable reduction in time.I heard that loading the graph is what takes time and hence I need to figure out how to load the graph only once and then feed the inputs without feeding all the parameters every time a new input needs to be tested.

The command that is used for decoding is as follows: 

python tensorflow/examples/speech_commands/label_wav.py \
--graph=/tmp/my_frozen_graph.pb \
--labels=/tmp/speech_commands_train/conv_labels.txt \
--wav=/tmp/speech_dataset/left/a5d485dc_nohash_0.wav

Also does anyone know if there is official data on how long alexa/google home take to respond to trigger words?

**Comment by drpngx** [comment]
@serendipity24 please note that you first have to load the tensorflow package. That takes about 2-3 seconds on python2, and in the 1.3 version, it can take 8 seconds (that is before we fixed the inspect stack in the tf decorator).

Running the inference should be very quick. You can modify the program to load, then ask for a prompt and pass in a waveform.

**Comment by serendipity24** [comment]
@drpngx Thank you for your reply. 

"You can modify the program to load, then ask for a prompt and pass in a waveform" - I will try this.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by drpngx** [comment]
Assuming this worked for you.
